src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,16 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery game = new Lottery();
+
+        Ball ball1 = game.getRandomBall();
```

Comment:
Think how to avoid creating balls this way. What if balls will be 100, we don't need to create 100 object line by line